### ISSUE TITLE
Guild info serialization config missing

### ISSUE THREAD

**Issue by waggz81** [Chalsean/XFaction#321]
We're effectively stuck on making any changes since we're missing the gui to edit the ginfo string.

**Comment by Chalsean** [comment]
yeah sorry, got distracted fixing other bugs.  you only need to use the serialization/compression logic if you settings is too big for the guild info.  is yours too many characters?

**Comment by waggz81** [comment]
With adding all the raid teams yeah it exceeded the limit with what we had when we initially configured it. I think we have room if that's all we put in the ginfo but even just having like discord info in there I think put us over, can't recall.

**Comment by Chalsean** [comment]
try 4.1.13, i dont have any way of testing myself unfortunately.  it likely will choke on the first load, as there was a switch in the code base in 4.0 to a better compression library.  you may have to manually enter the first go

**Comment by waggz81** [comment]
There's nothing I can see in the options to redo the serialization unless I'm blind.
![image](https://user-images.githubusercontent.com/20745474/202665665-3ae96b33-1dbb-480a-a863-b39b11528519.png)

This is what I get from debuglog:
```
58;11-18 03:11:37;JXFa:Champion;3;`Alt^^>;
59;11-18 03:11:37;TimerEvent;1;...face/AddOns/XFaction/Confederate/GuildCollection.lua:134: Player is not on a supported guild or realm;
60;11-18 03:11:37;TimerEvent;1;[string "@Interface/AddOns/XFaction/Developer/Logger.lua"]:29: in function `Error';
61;11-18 03:11:37;TimerEvent;1;[string "@Interface/AddOns/XFaction/Handlers/TimerEvent.lua"]:95: in function `catchBlock';
62;11-18 03:11:37;TimerEvent;1;[string "@Interface/AddOns/XFaction/Functions/Exception/try-catch-finally.lua"]:30: in function `catch';
63;11-18 03:11:37;TimerEvent;1;[string "@Interface/AddOns/XFaction/Handlers/TimerEvent.lua"]:94: in function <Interface/AddOns/XFaction/Handlers/TimerEvent.lua:21>;
64;11-18 03:11:37;TimerEvent;1;
```

**Comment by waggz81** [comment]
I think I recreated the config and we have 10 chars left, with the only other thing being the discord server link. Provided I haven't missed anything or we need to add another team, so can probably work around this for now. 320 with the string vs 449 longform, and it's a 499 cap, we're at 489 with the discord info. Would be nice to be able to add website in there too.

**Comment by Chalsean** [comment]
sorry forgot to checkin one of the files i updated.  try 4.1.13 now

**Comment by waggz81** [comment]
Okay that one seemed to work. I was able to take the recreated config I wrote out last night and use the options screen to serialize it and the new version reads it correctly.

**Comment by waggz81** [comment]
fixed in [v4.1.14](https://github.com/Chalsean/XFaction/releases/tag/v4.1.14) ([a3d9298](https://github.com/Chalsean/XFaction/commit/a3d9298f9ba29143864b23ebf3e2eb4b1cd52423))
